config/locales/en/management.yml

```diff
@@ -121,6 +121,8 @@ en:
       erase_account_confirm: Are you sure you want to erase the account? This action can not be undone
       erase_warning: This action can not be undone. Please make sure you want to erase this account.
       erase_submit: Delete account
+      edit_account: Edit account
```

Comment:
As the email is the only field we're changing here...I would go for an "Edit email" and "Update email"